microsoft-edge/webview2/release-notes/index.md

```diff
@@ -218,8 +220,10 @@ The following APIs have been promoted to Stable and are now included in this Rel
 ---
 
 
-<!-- ------------------------------ -->
-* Added `SaveAs` APIs that allow you to programmatically perform the **Save as** operation.  You can use these APIs to block the default **Save as** dialog, and then either save silently, or build your own UI for **Save as**.  These APIs pertain only to the **Save as** dialog, not the **Download** dialog, which continues to use the existing Download APIs.
+<!-- ---------- -->
+###### SaveAs
```

Comment:
best wording for heading?

see rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3253&tabs=dotnetcsharp#saveas

Establishing new, improved formatting/presentation.
@victorhuangwq @peiche-jessica

Reply:
making the heading more verbose and specific could be good, in case different releases add different SaveAs functionality.

such heading text is linkable as a link destination, eg
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#saveas
which is somewhat unstable; liable to change to #saveas-1 or #saveas-2 to scroll to that exact section, if a newer heading w/ same wording is added later eg when an API entry moves from Experimental in Prerelease, to Stable in Prerelease, to Stable in Release.  

But that type of ambiguity is fine, b/c normally you want the latest, topmost entry, anyway.  
The greater concern or need is, differentiate entries that add different SaveAs functionality (rather than merely moving the same new functionality through the stages pipeline).

Reply:
closing, b/c  entered a Suggestion Comment to list some alternative, more verbose headings.